### DISCUSSION TITLE
Can't get access to copilot even if I have Student Pack already

### DISCUSSION THREAD

**Discussion by AperturePlus** [community/community#115486] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

I've already applied for Student Pack and got approved since 9/2023. Several days ago I can still chat with copilot. However today my IDEs(vscode, IDEA and so on) tell me that I dont have the access to copilot and tell me to contact support. I wonder if there're any bugs. 
:(

**Comment by AperturePlus** [comment]
![image](https://github.com/community/community/assets/146049978/9e23b497-74fc-4ed0-af21-d474c021a4a6)
![image](https://github.com/community/community/assets/146049978/c03257e6-1717-49a4-8dbe-925e9f4c9e4b)

**Comment by AperturePlus** [comment]
[info] [auth] Invalid copilot token: missing token: 403 
2024-03-31 17:43:43.187 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as AperturePlus."

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @AperturePlus ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by AperturePlus** [reply]
thank u :)

**Comment by samus-aran** [comment, marked as answer]
Hi @AperturePlus 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
